src/main/java/com/kodilla/ecommercee/config/security/MyUserDetailService.java

```diff
@@ -0,0 +1,36 @@
+package com.kodilla.ecommercee.config.security;
+
+import com.kodilla.ecommercee.entity.User;
+import com.kodilla.ecommercee.repository.UserRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.security.core.GrantedAuthority;
+import org.springframework.security.core.authority.SimpleGrantedAuthority;
+import org.springframework.security.core.userdetails.UserDetails;
+import org.springframework.security.core.userdetails.UserDetailsService;
+import org.springframework.security.core.userdetails.UsernameNotFoundException;
+import org.springframework.stereotype.Service;
+
+import java.util.HashSet;
+import java.util.Set;
+
+@Service
+public class MyUserDetailService implements UserDetailsService {
+
+    @Autowired
+    UserRepository userRepository;
```

Comment:
Tutaj wydaje mi się, że zgodnie z ostatnią rozmową powinien być modyfikator dostepu.

Reply:
Zgadza się, to tak z rozpędu ...